docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+
+The most recent successful backup set name can be referenced in one of the following ways:
+- `:latest` - The latest backup set name among all backup groups.
+- `:group-latest` - The latest backup set name for the backup group in current database deployment.
+
+Two restore/import source types are available:
+- _stream_ - Used as a synonym for a copy of NuoDB archive which contains both the archive and journal directories.
+A remote _stream_ source is downloaded and extracted directly into the archive directory which doesn't require extra space for holding the restore/import source file.
+- _backupset_ - Hot copy backup set which is either fetched from a remote source or available in the backup directory.
+
+### Distributed database in-place restore
+
+Database in-place restore can recover from a complete loss or data corruption of all database archives by reverting the database state to a previous restore point.
+NuoDB _restore_ chart is used to **overwrite** the existing database state using a configured restore source.
+To perform a database in-place restore of the entire database, the `restore.type` setting should be set to `database` and the database must be shut down and restarted.
+NuoDB will ensure that on the restart, the SMs serving restored archives are started first, and all other database processes will wait for the database restore to complete.
+Archives that are not restored will then be synced from the running SM processes.
+
+> **NOTE**: If the restore source was copied from another environment, then all archives need to be restored. Otherwise, the Storage Manager process will fail to start with error _Archive "/var/opt/nuodb/archive/nuodb/demo" doesn't match the database.  Expected UUID \*\*\*, got \*\*\*._
+An alternative method is to create a new database deployment using [Automatic archive initial import](#automatic-archive-initial-import).
+If part of doing this involves destroying an old environment, be sure to clean up the old storage volumes to avoid additional storage costs.
+
+The database restart during the restore process is controlled by the `restore.autoRestart` setting value (by default _"true"_).
+Users can retain control to manually stop and restart the database pods by disabling the auto-restart during the _restore_ chart installation.
+All SMs serving restored archives will need to be started together so that the database restore process can complete.
+
+The high-level steps to perform database in-place restore are the following:
+
+1. Identify the restore source which can be either a backup set created by scheduled backup jobs or a remote source.
+2. Identify which archives need to be restored.
+By default, all archives served by HC SMs in a single Kubernetes cluster will be selected for a restore as they can define the database state by restoring from a backup set available in the backup volumes.
+3. Ensure that the restore source is available on all SMs selected for restore.
```

Comment:
I would add a note saying that if the restore source is a backup set, then only HC SMs will have access to the restore source.